tests/e2e/standard/test_base.py

```diff
@@ -140,6 +140,7 @@ def test_delete_namespace(self, container: MetadataContainer):
 
 class TestStringSet(BaseE2ETest):
     neptune_tags_path = "sys/tags"
+    neptune_group_tags_path = "sys/group_tags"
```

Comment:
`sys/group_tags` may not yet be available in production environment. Would you mind to add some environment variable? Something similar to lightning ecosystem probably.